.github/workflows/sync_translations.yml

```
@@ -27,6 +27,11 @@ jobs:
             curl -OL https://github.com/transifex/cli/releases/download/v1.6.17/tx-linux-amd64.tar.gz
             tar -xvzf tx-linux-amd64.tar.gz
 
+      - name: Export the env variables file
```

```suggestion
      - name: Copy .env file
```

If you do not export, better name the step like this.